### DISCUSSION TITLE
为什么标题栏只能输入特定长度

### DISCUSSION THREAD

**Discussion by Ad-closeNN** [Meloong-Git/PCL#3394] [提问] [answered]
如题，为什么启动器的标题栏只能塞下到如图所示到`|`的长度
~~`易崩溃™启动器 || Ultral Pro Max Plus`刚好塞的下~~
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/2d96773c-c2ec-4bea-b5e4-962949a96bb8)

**Comment by allMagicNB** [comment, marked as answer]
太长了会被其他按钮挡住，如果用“…”替代多余的部分也还是没有显示出来

**Reply by Ad-closeNN** [reply]
那我执意要突破怎么办.jpg

**Reply by MoYuan-CN** [reply]
没办法的..

**Reply by allMagicNB** [reply]
> 那我执意要突破怎么办.jpg

那你去改ini喽

**Reply by Ad-closeNN** [reply]
> > 那我执意要突破怎么办.jpg
> 
> 那你去改ini喽

~~还真行~~
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/0fe2269f-fab8-4094-916e-b2652f9fe54e)
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/f5a209df-8b72-475e-99f6-3a655d7edacb)

**Reply by Chiloven945** [reply]
太长不是挡住按钮是被按钮挡住……
